src/plugins/template/src/sync_infer_request.cpp

```diff
@@ -269,8 +269,12 @@ std::vector<ov::ProfilingInfo> ov::template_plugin::InferRequest::get_profiling_
         return p_info;
     };
     info.emplace_back(fill_profiling_info("input preprocessing", m_durations[Preprocess]));
-    info.emplace_back(fill_profiling_info("execution time", m_durations[StartPipeline]));
+    auto template_model = get_template_model();
+    for (const auto& op : template_model->get_runtime_model()->get_ops()) {
+        info.emplace_back(fill_profiling_info(op->get_friendly_name(), m_durations[StartPipeline]));
```

Comment:
is it correct to set the same time for each operation?

Reply:
I’m not sure how important the reliability of the information is here, but thank you, now I’ve changed it and the time is taken from the perf counter